panel/src/components/Forms/Input/ListInput.vue

```diff
@@ -2,33 +2,25 @@
 	<k-writer
 		ref="input"
 		v-bind="$props"
-		:extensions="extensions"
+		:extensions="listExtensions"
 		:value="list"
 		class="k-list-input"
 		@input="onInput"
 	/>
 </template>
 
 <script>
-import { disabled } from "@/mixins/props.js";
+import { props as WriterInputProps } from "@/components/Forms/Input/WriterInput.vue";
 import ListDoc from "@/components/Forms/Writer/Nodes/ListDoc.js";
 
 export const props = {
-	mixins: [disabled],
+	mixins: [WriterInputProps],
```

Comment:
Naming is a bit odd, I think in other cases we would just name this `WriterInput`

Reply:
E.g. https://github.com/getkirby/kirby/blob/v4/lab/inputboxes/panel/src/components/Dialogs/FormDialog.vue#L23 it's just `Fields` not `FieldsProps`

Reply:
I agree that it should be consistent. But we are actually really just importing the props as mixin here. I really think that the "WriterInput" version is more confusing to me than the more verbose variant.